integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/freeipa/FreeIpaUpgradeTests.java

```diff
@@ -268,6 +270,20 @@ private void generateHostKeyTab(com.sequenceiq.freeipa.api.client.FreeIpaClient
         }
     }
 
+    private void generateServiceKeytab(com.sequenceiq.freeipa.api.client.FreeIpaClient ipaClient, String environmentCrn, String accountId) {
+        try {
+            ServiceKeytabRequest serviceKeytabRequest = new ServiceKeytabRequest();
+            serviceKeytabRequest.setEnvironmentCrn(environmentCrn);
+            serviceKeytabRequest.setServiceName("test");
+            serviceKeytabRequest.setServerHostName("test.local");
+            serviceKeytabRequest.setDoNotRecreateKeytab(Boolean.FALSE);
+            ipaClient.getKerberosMgmtV1Endpoint().generateServiceKeytab(serviceKeytabRequest, accountId);
```

Comment:
`accountId` shouldn't be set here, as the test framework is not using an internal actor to call services. could you use `null` here and see if it works? (it should)

Reply:
Done. It works with null.